felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -3099,7 +3100,13 @@ func strToByte64(s string) [64]byte {
 }
 
 func (m *bpfEndpointManager) ruleMatchIDFromNFLOGPrefix(nflogPrefix string) polprog.RuleMatchID {
-	return m.lookupsCache.GetID64FromNFLOGPrefix(strToByte64(nflogPrefix))
+	if m.lookupsCache != nil {
+		return m.lookupsCache.GetID64FromNFLOGPrefix(strToByte64(nflogPrefix))
+	}
+	// Lookup cache is not available, so generate an ID out of provided prefix.
+	h := fnv.New64a()
+	h.Write([]byte(nflogPrefix))
+	return h.Sum64()
```

Comment:
This is the approach we had before introducing flow logs.

Reply:
https://github.com/projectcalico/calico/blob/232b72fce37b6dfff316e3977efba27fc5058696/felix/dataplane/linux/bpf_ep_mgr.go#L4159

Reply:
Do we use this for anything when flow logs are disabled?

Reply:
Yes, we do.